### PR TITLE
[REVIEW PR] Release 1.14.5 -> 1.14.4

### PR DESCRIPTION
**Pull request by ntn-x2**
Changes compared to the last release for this PR.

### CODE REVIEW

File: pallets/attestation/src/tests/delete.rs [hunk 1]
```diff
@@ -37,57 +37,9 @@ fn test_remove() {
...
-	ExtBuilder::default()
-		.with_balances(vec![(ACCOUNT_00, <Test as Config>::Deposit::get() * 100)])
-		.with_ctypes(vec![(attestation.ctype_hash, attester.clone())])
-		.with_attestations(vec![(claim_hash, attestation)])
-		.build_and_execute_with_sanity_tests(|| {
-			assert_ok!(Attestation::remove(
-				DoubleOrigin(ACCOUNT_00, attester.clone()).into(),
-				claim_hash,
-				None
+				authorization_info
```
Comment (Ad96el) [inline pallets/attestation/src/tests/delete.rs:40]: Release 1.14.4 contains the broken attestation events?

Comment (Ad96el) [reply pallets/attestation/src/tests/delete.rs:40]: Ah, wait 1.14.4 only contained fixes for the node right?

Comment (ntn-x2) [reply pallets/attestation/src/tests/delete.rs:40]: This looks like a GitHub bug. The two files are exactly the same if you open them on each respective branch. They also don't show as diff in my VS Code. Uh, that makes the whole point of this review kinda worthless. Maybe I can try to adjust the history, let's see.

Comment (ntn-x2) [reply pallets/attestation/src/tests/delete.rs:40]: I would suggest you look at the PR from VS Code, which seems to be better at showing actual diffs.

**Review by Ad96el — APPROVED** [review]

**Pull request closed without merging**